external-crates/move/crates/move-compiler/tests/move_2024/unit_test/random_test_invalid.snap

```diff
@@ -5,4 +5,8 @@ info:
   edition: 2024.alpha
   lint: false
 ---
-
+error[E02015]: invalid attribute
+  ┌─ tests/move_2024/unit_test/random_test_invalid.move:7:7
+  │
+7 │     #[random_test(b = 1)]
+  │       ^^^^^^^^^^^^^^^^^^ Attribute 'random_test' does not support parameters. Expected '#[random_test]' with no arguments
```

Comment:
We should maybe expand this note to specify that the values will be randomly generated